style.css

```diff
@@ -119,7 +119,7 @@
   font-family: "Poppins", sans-serif;
   font-style: normal;
   font-weight: 700;
-  font-size: 20px;
+  font-size: 18px;
```

Comment:
### NOTE: This change is not applicable to this line of code. I had to use it to add an inline comment. please refer to the review comment to get clarity on changes to make below: :pray: 

- [ ] Let's add some padding-top to the `.intro` to add some spaces between the header. You can try `padding-top: 50px` or something else to improve the design.

- [ ] Let's also remove the `/* height: 122px;` from the `intro-paragraph h1` as this is not needed so that our design is better.
- [ ] Let's make the `line-height: 24px;` as this is the correct value used to improve our design.

Reply:
Fixed.